gnd/src/main/java/com/google/android/gnd/ui/common/BindingAdapters.java

```diff
@@ -118,4 +126,9 @@ public static void setOnFocusChangeListener(
   public static void setOnShowDialogListener(MultipleChoiceFieldLayout view, Runnable listener) {
     view.setOnShowDialogListener(listener);
   }
+
+  @BindingAdapter("imageUri")
+  public static void bindUri(ImageView view, Uri uri) {
+    Picasso.get().load(uri).placeholder(R.drawable.ic_photo_grey_600_24dp).into(view);
```

Comment:
Do we need to handle error states here? Will failing to load the URI cause the application to crash?

Reply:
No, that's why we have a placeholder. If anything unexpected happens, the app doesn't crash and the imageview just displays placeholder.

I tested network not-available and null cases

Reply:
Great, thanks for checking! Could you kindly add a comment for posterity since it's not obvious at first glance?